.travis.yml

```diff
@@ -1,13 +1,35 @@
 language: java
 sudo: required
+dist: trusty
+#group: edge
+
 script:
   - ./gradlew check
 cache:
   directories:
   - $HOME/.gradle
-jdk:
-  - oraclejdk8
-  - openjdk6
+
+before_install:
+  - export GRADLE_OPTS=-Xmx1024m
+
+matrix:
+  include:
+    - jdk: openjdk7
```

Comment:
Not much option here I think, JDK 6 is less and less supported by Gradle/Travis. As long as the compile target is 6, we should be okay.